### ISSUE TITLE
Visual Test for crosshairs not working as designed

### ISSUE THREAD

**Issue by jdunkerley** [ScottLogic/d3fc#637]
In playing with the css for the multi series I noticed the visual test for crosshairs is trying to set the colour of the paths:

```javascript
        .decorate(function(sel) {
            sel.style('fill', function(d) { return color(d.date.getDay()); });
        })
```

The style gets applied on the g rather than the path within it.

The current css applies the default colour on .bar>path.

Could be fixed by change the css to apply to g.bar instead. The change to a series class I was trying for the multi series also fixed it.

Old output:
![image](https://cloud.githubusercontent.com/assets/4699705/10590662/d7dc9d06-76a9-11e5-85a5-855ec7bcf5b8.png)

Expected output:
![image](https://cloud.githubusercontent.com/assets/4699705/10590687/f7e48cbc-76a9-11e5-837d-1e6308c1a0b8.png)

**Comment by jdunkerley** [comment]
Alternatively could change the visual tests to decorate the path with the fill:
```javascript
        .decorate(function(sel) {
            sel.select('path').style('fill', function(d) { return color(d.date.getDay()); });
        })
```

**Comment by chrisprice** [comment]
This looks like a hangover from the original implementation of the bar
series that was not updated when the series was updated. I suppose the fix
depends upon the resolution of the #629.

**Comment by jdunkerley** [comment]
This can be closed as fixed by #642

**Comment by ColinEberhardt** [comment]
Thanks @jdunkerley
